eating_cookies/eating_cookies.py

```diff
@@ -3,14 +3,38 @@
 import sys
 
 # The cache parameter is here for if you want to implement
-# a solution that is more efficient than the naive 
+# a solution that is more efficient than the naive
 # recursive solution
+# 0 is irrelivant. There are 0 ways to eat 0 cookies.
+# Same goes for neg numbers.
+# Should take in integer, and return integer.
+
+
 def eating_cookies(n, cache=None):
-  pass
+    if n == 0:
+        return 1
+    elif n < 0:
+        return 0
+
+    # Number of cookies cookie monster can eat is:
+    # 0, 1, 2, 3. Ignore 0 - it doesn't do anything.
+    num_ways = 0
+    for i in range(1, 4):
```

Comment:
I saw this portion in you commit messages - you can set this so that it is dependent on the length of the array passing through. 
Something like `for i in range(0, len(arr))` then increment `i` in the code.